images/cilium/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG CILIUM_RUNTIME_IMAGE=quay.io/cilium/cilium-runtime:f0c4a2c2ce4af34136027dbe1
 
 
 # cilium-envoy from github.com/cilium/proxy
-#
+# If envoy version is updated, please update related matrix (e.g. envoy.rst) as well.
```

Comment:
I'd use the “full” path (Documentation/security/network/proxy/envoy.rst) to help people find the doc.